src/genie/libs/parser/iosxe/show_wireless.py

```diff
@@ -2053,3 +2053,100 @@ def cli(self, output=None):
                 wireless_info_obj["base_mac"].update(ap_info_dict)
 
         return wireless_info_obj
+
+
+# ======================================
+# Schema for:
+#  * 'show wireless fabric vnid mapping'
+# ======================================
+class ShowWirelessFabricVnidMappingSchema(MetaParser):
+    """Schema for show wireless fabric vnid mapping."""
+
+    schema = {
+      Optional("fabric_vnid_mapping") : {
+        Optional("name"): {
+          Optional(str) : {
+            Optional("l2_vnid"): int,
+            Optional("l3_vnid"): int,
+            Optional("ip_address"): str,
```

Comment:
I think `ip_address` is only `Optional`? based on given command output for test.

Reply:
@tahigash The open issue [here](https://github.com/networktocode/genieparser/issues/119) has examples where there is no output. That is why I put Optional for all the keys.

Reply:
In case of no output, `fabric_dict` should be empty as return. and parser will raise `SchemaParserError`. so don't need to change all to Optional and let's have Optional for item which is needed.

Reply:
@tahigash thanks for letting me know. I have updated to include `Optional()` only for `ip_address`